source/guides/1.1/getting-started.md

```diff
@@ -980,7 +976,7 @@ end
 ```
 
 In our template we can loop over `params.errors` (if there are any) and display a friendly message.
-Open up `apps/web/templates/books/new.html.erb`:
+Open up `apps/web/templates/books/new.html.erb` and add the following at the top of the file:
```

Comment:
Up until now we always presented the entire file. Here we only show a piece of it. Maybe we should just paste the entire file here as well?

Reply:
:+1: